hardware/camera/Android.mk

```diff
@@ -1,19 +1,31 @@
+# Create symlinks for camera libraries. Currently only applicable
+# to sdm845 boards.
+
+LOCAL_PATH := $(call my-dir)
+
 ifneq ($(filter sdm845,$(TARGET_BOARD_PLATFORM)),)
 
-$(shell mkdir -p $(PRODUCT_OUT)/$(TARGET_COPY_OUT_ODM)/lib)
-$(shell mkdir -p $(PRODUCT_OUT)/$(TARGET_COPY_OUT_ODM)/lib/camera)
-$(shell mkdir -p $(PRODUCT_OUT)/$(TARGET_COPY_OUT_ODM)/lib64)
-$(shell mkdir -p $(PRODUCT_OUT)/$(TARGET_COPY_OUT_ODM)/lib64/camera)
```

Comment:
I don't understand why there were mkdir commands for the odm partition here.  Seems like a typo.  Hopefully, they were not important for odm-image builds :-)

Reply:
Good find, I didn't even notice those were for `odm`. Same [applies to Pie](https://github.com/sonyxperiadev/device-sony-common/pull/552/files#diff-4319f036f6f92ccd772cfac4ff641a73L3) btw.

Currently checking with @jerpelea how we can adjust odm building on his side without depending on unrelated makefiles to run.